services/search/utils.py

```diff
@@ -0,0 +1,31 @@
+import requests # pylint: disable=import-error
+import constants
+
+def create_synonym_list_esp(word_arr, max_synonyms=5):
+    """
+    Retrieves a synonym list given a word list
+    Attributes:
+        word_arr: The word list which we wish to find synonyms for
+        max_synonyms: The maximum number of synonyms per word in word_arr
+            optional: Default value 5
+    Returns: a list of synonyms
+    """
+    synonyms = []
+    for word in word_arr:
+        try:
+            payload = requests.get(constants.SPANISH_API_URL+word)
+            payload.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            return {"error": {"type": "HTTPError", "status_code": err.response.status_code,
+                "message": getattr(err, 'message', str(err))}}
+        except requests.exceptions.RequestException as e:
```

Comment:
Same as with `utils.py` from `keywords/`, you should re-raise the exception and defer formatting the error message to the calling function.